combat-trainer.lic

```diff
@@ -2259,16 +2261,42 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.aiming_trainables
+        skill = game_state.determine_aiming_skill
+
+        if skill.eql?('Brawling')
+          3.times{ return if !execute_aiming_action?('gouge', game_state) }
+          return
+        elsif skill.eql?('Tactics')
+          %w[bob weave circle].each do |action|
+            return if !execute_aiming_action?(action, game_state)
+          end
+          return
+        end
+
+        weap = game_state.wield_offhand_weapon(skill)
+        if skill.include?('Thrown')
+          actions = %w[lob lob lob]
+        else
+          actions = %w[jab feint draw]
```

Comment:
💡 If we're using weak attacks, we only `jab` (see line 3097).  Do we want to `feint` and `draw` here? Or have a conditional on `use_weak_attacks`?

Reply:
Yeah, I actually have a modified version of weak attacks which does a sample of jab or feint. I know Leu does Draw on his so I have this setup. What I can do is jab/feint/draw unless weak attacks, then jab only?